04/app.js

```diff
@@ -3,6 +3,37 @@ document.addEventListener('DOMContentLoaded', init);
 function init() {
     const boxElement = document.querySelector('.box');
     setBoxShadow(boxElement, '#000000');
+
+    const colorElement = document.querySelector("[type='color']");
+    const rangeElement = document.querySelector("[type='range']");
+
+    colorElement.addEventListener("change", changeColor);
+    rangeElement.addEventListener("change", changeOpacity);
+    rangeElement.addEventListener("mousemove", changeOpacity);
```

Comment:
To chyba za mało, aby działało prawidłowo. Jak zmienię `opacity` przez kliknięcie, a potem najadę na ten element to tracę odpowiednie ustawienia.